web/docs/tql2/operators/chart_area.md

```diff
@@ -0,0 +1,56 @@
+# chart_area
+
+Chart values on a X-Y area chart.
+
+```tql
+chart_area x=any, y=any, [x_min=any, x_max=any, resolution=duration, x_log=bool, y_log=bool, group=any, position=string]
+```
+
+## Description
+
+Chart values on X-Y area chart.
+
+### `x = any`
+
+The values to plot on the X-axis.
```

Comment:
Should we use "x-axis" instead of "X-axis"? Both can be used, but since we lowercase `x` in the parameters, I feel like we should also lowercase it in the description.

Reply:
Then let's go with lowercase then. I think the consistency with the parameter names is nice to have.

Reply:
Changed to lowercase.